Wikipedia/Code/WebViewController.m

```diff
@@ -290,8 +290,7 @@ - (WMFArticleFooterView*)footerLicenseView {
         @weakify(self);
         [_footerLicenseView.showLicenseButton bk_addEventHandler:^(id sender) {
             @strongify(self);
-            MWKSite* site = [[MWKSite alloc] initWithDomain:WMFDefaultSiteDomain language:@"en"];
-            [self.delegate webViewController:self didTapOnLinkForTitle:[site titleWithString:WMFLicenseTitleOnENWiki]];
```

Comment:
@josve05a I tried it. No it doesn't work. Out of curiosity, what was the problem with the former page this was linking to?